firebase.json

```diff
@@ -1,6 +1,6 @@
 {
   "database": {
-    "rules": "./database.rules.json"
+    "rules": "rules.bolt"
```

Comment:
Shouldn't we add a `firebase-bolt` to the dev dependencies?

Reply:
I think it's preferred to be a global install.